components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -39,13 +40,13 @@ namespace {
 using ArticleInfo = std::tuple<mojom::FeedItemMetadataPtr, Signal, double>;
 using ArticleInfos = std::vector<ArticleInfo>;
 
-const Signal* GetSignal(const mojom::FeedItemMetadataPtr& article,
-                        const Signals& signals) {
-  auto it = signals.find(article->url.spec());
+const Signal GetSignal(const mojom::FeedItemMetadataPtr& article,
```

Comment:
nit: I think we can drop `const` from the return type as we're cloning and it'd go well with `auto` from the callsite